src/seeds/seeder.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { Users } from '../users/users.model';
+import { NewUser } from '../types/users';
+
+@Injectable()
+export class SeederRepository {
+  constructor(
+    @Inject(Users.name)
+    private readonly userModel: typeof Users,
+  ) {}
+
+  checkDataUsers(id: number): Promise<boolean> {
+    return this.userModel
+      .findOne({
+        where: { id },
+      })
+      .then((res) => res !== null);
+  }
+
+  createUsers(users: NewUser[]): Promise<NewUser[]> {
```

Comment:
as all methods in the service is already scoped by class itself, no need to add `Users` to method names. Just `create`

Reply:
Just make it a private method)